.travis/install-secp256k1.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -eux
+
+gitRef="1086fda4c1975d0cad8d3cad96794a64ec12dca4"
+
+curl -LO "https://github.com/bitcoin-core/secp256k1/archive/$gitRef.zip"
+
+unzip "$gitRef.zip"
+
+cd "secp256k1-$gitRef"
+
+./autogen.sh
+# hevm needs reecovery module
+# don't install shared library so linker must used static
+# enable pic so static library can link against dynamic correctly
+./configure --enable-module-recovery --enable-shared=no --with-pic
+sudo make install
```

Comment:
README instructions should be updated explain how the installation process should be performed. Keep in mind that we don't want to recommend people to install libraries using `sudo make install` since it's very difficult to un-install them later (for travis scripts, it's ok, we don't care).

Reply:
I added a bunch of instructions to the README to cover this. Take a look and let me know if you want me to make any changes.